scapy/layers/tls/cert.py

```diff
@@ -279,9 +279,9 @@ def fill_and_store(self, modulus=None, modulusLen=None, pubExp=None):
     def import_from_tuple(self, tup):
         # this is rarely used
         e, m, mLen = tup
-        if isinstance(m, str):
+        if isinstance(m, (str, bytes)):
```

Comment:
I think `isinstance(m, bytes)` should be enough.

Reply:
Updated